src/commands/deploy.ts

```diff
@@ -600,3 +609,31 @@ function getCloudProvider(provider: string): CloudAdapter {
             throw new Error(`Unsupported cloud provider: ${provider}`);
     }
 }
+
+// If the cloud provider defined in the configuration file is `genezio`,
+// we will randomly change it to in order to test out our experimental runtime.
+export async function performCloudProviderABTesting(
+    configuration: YamlProjectConfiguration,
+): Promise<YamlProjectConfiguration> {
+    // Skip the AB testing if the project is already deployed
+    const isAlreadyDeployed = await isProjectDeployed(configuration.name, configuration.region);
+
+    // We won't perform AB testing if the cloud provider is set for runtime, self-hosted or cluster.
+    if (!isAlreadyDeployed && configuration.cloudProvider === CloudProviderIdentifier.GENEZIO) {
+        const randomCloudProvider = getRandomCloudProvider();
+
+        if (randomCloudProvider !== CloudProviderIdentifier.GENEZIO) {
+            debugLogger.debug(
+                "You've been visited by the AB testing fairy! 🧚‍♂️ Your cloud provider is not set to",
```

Comment:
i think there is a typo here. `now` instead of `not`

Reply:
Uuu, great catch! Thanks